x-pack/test/functional/apps/spaces/spaces_selection.ts

```diff
@@ -127,7 +127,8 @@ export default function spaceSelectorFunctionalTests({
       });
     });
 
-    describe('Spaces Data', () => {
+    describe('Spaces Data', function () {
+      this.tags('skipFirefox');
```

Comment:
We are limited with FF slowness right now, I suggest we stick to the minimum tests on FF until we find a solution.